src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h

```diff
@@ -49,18 +49,18 @@ class FlybyPoint: public SysModel {
     void SelfInit();                   //!< Self initialization for C-wrapped messages
     void Reset(uint64_t CurrentSimNanos);
     void UpdateState(uint64_t CurrentSimNanos);
-    
-    double     dtFilterData = 0;       //!< time between two subsequent reads of the filter information
-    double     epsilon = 0;            //!< tolerance for singular conditions when position and velocity are collinear
-    int64_t    signOfOrbitNormalFrameVector = 1;  //!< Sign of orbit normal vector to complete reference frame
-    FlybyModel flybyModel;             //!< flag to indicate which flyby model is being used
 
     ReadFunctor<NavTransMsgPayload>  filterInMsg;               //!< input msg relative position w.r.t. asteroid
     ReadFunctor<EphemerisMsgPayload> asteroidEphemerisInMsg;    //!< input asteroid ephemeris msg
     Message<AttRefMsgPayload> attRefOutMsg;                     //!< Attitude reference output message
     AttRefMsg_C attRefOutMsgC = {};                             //!< C-wrapped attitude reference output message
 
 private:
+    double     timeBetweenFilterData = 0;       //!< time between two subsequent reads of the filter information
+    double     toleranceForCollinearity = 0;            //!< tolerance for singular conditions when position and velocity are collinear
+    int64_t    signOfOrbitNormalFrameVector = 1;  //!< Sign of orbit normal vector to complete reference frame
+    FlybyModel chosenFlybyModel = rectilinear;              //!< enum to indicate which flyby model is being used
+
     bool            firstRead;           //!< variable to attest if this is the first read after a Reset
```

Comment:
:large_orange_diamond: We should class initialize these also. E.g. 
`bool firstRead=true;` and so on

Reply:
done